tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/send_build_to_bugsnag.rb

```diff
@@ -47,6 +46,15 @@ def self.run(params)
         if payload[:appVersion].nil?
           UI.user_error! missing_app_version_message(params)
         end
+
+        # Is the Fastlane verbose flag enabled? store local flag
+        verbose = UI.verbose("Sending build to Bugsnag with payload:")
```

Comment:
No need to store the local flag imo. Clearer to just let fastlane track that and log at verbose level regardless.

Reply:
Fixed in https://github.com/bugsnag/bugsnag-dsym-upload/pull/42/commits/0ee4f61e6d19204002d22745ef6c2a7e29804ea0